docs/source/actionchain.rst

```diff
@@ -9,39 +9,55 @@ Authoring an ActionChain
 ActionChain's are described in YAML (JSON supported for backward compatibiltiy) and placed inside a pack similar to other script or python actions. An ActionChain must also be associated with a metadata file that allows it to be registered as an Action by |st2|. This metadata contains name and parameter description of an action.
 
 
-ActionChain script
-~~~~~~~~~~~~~~~~~~
+ActionChain definition
+~~~~~~~~~~~~~~~~~~~~~~
 
 Following is sample ActionChain workflow definition named :github_st2:`echochain.yaml
-<contrib/examples/actions/echochain.yaml>`:
+<contrib/examples/actions/workflows/echochain.yaml>`:
 
-.. literalinclude:: /../../contrib/examples/actions/echochain.yaml
+.. literalinclude:: /../../contrib/examples/actions/workflows/echochain.yaml
    :language: yaml
 
-Note:
+Details:
 
 * `chain` is the array property that contains action elements.
 * Action elements are named action execution specifications. The name is scoped to an ActionChain and is used as a reference to an action element.
 * `ref` property of an action element points to an Action registered in |st2|.
 * `on-success` is the link to action element to invoke next on a successful execution. If not provided the ActionChain will terminate with status set to success.
 * `on-failure` is the link to action element to invoke next on a failed execution. If not provided the ActionChain will terminate with the status set to error.
-* `default` is the top level property that specifies start of an ActionChain.
+* `default` is an optional top level property that specifies the start of an ActionChain.
```

Comment:
Should probably make it clear and say something along the lines of "if default is not specified, first action in the chain is considered as default and executed first".